### PR TITLE
Added paragraph of text.

### PR DESCRIPTION
**Pull request by owenkcraig**
- Added lorem impsum text as a paragraph.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,5 +8,6 @@
 </head>
 <body>
     <h1>That's a good cat right there.</h1>
+    <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Aliquid laborum officia beatae, perferendis vitae esse voluptate laboriosam velit nesciunt quas, ad, voluptas necessitatibus? In provident modi, fugiat eum mollitia molestiae.</p>
```
Comment (EstherEdell) [inline index.html]: this looks great, but can we remove the last sentence? it is unnecessary.

**Review by EstherEdell — CHANGES_REQUESTED** [review]
Looks good, just a couple things to clean up!

File: index.html (hunk 1 above)
Comment (owenkcraig) [reply index.html]: I have removed the bad sentence.

**Comment by owenkcraig** [comment]
Made requested changes.

**Review by EstherEdell — APPROVED** [review]
all good! go ahead and merge!!

**Pull request merged**